topics/introduction/tutorials/galaxy-intro-101-everyone/tutorial.md

```diff
@@ -0,0 +1,497 @@
+  - "Galaxy can connect to external sources for data import and visualization purposes"
+  - "Galaxy provides ways to share your results and methods with others"
+contributors:
+  - annefou
+  - nagoue
+  - chrisbarnettster
+  - michelemaroni89
+  - olanag1
+  - tnabtaf
+---
+
+
+# Introduction
+{:.no_toc}
+
+<!-- This is a comment. -->
+
+This practical aims to familiarize you with the Galaxy user interface. 
+It will teach you how to perform basic tasks such as importing data, running tools, working with histories, and sharing your work.
+Not everyone has the same background and that's ok! 
+
+{% include snippets/warning_results_may_vary.md %}
+
+> ### Agenda
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+> ### {% icon comment %} Background
+> The Iris flower data set or Fisher’s Iris data set is a multivariate dataset introduced by the British statistician and biologist Ronald Fisher in his 1936 paper ({% cite Fisher1936 %}). 
+> Each row of the table represents an iris flower, including its species and dimensions of its botanical parts, sepal and petal, in centimeters.
+> For more history of this dataset read here [Wikipedia](https://en.wikipedia.org/wiki/Iris_flower_data_set).
+{: .comment}
+
+## What does Galaxy look like?
+
+> ### {% icon hands_on %} Hands-on: Log in or register
+> Browse to your [Galaxy instance](https://galaxyproject.org/use/) and log in or register.
+> 1. Open your favorite browser (Chrome, Safari or Firefox as your browser, not Internet Explorer!)
+> 2. Browse to your Galaxy instance
+> 3. Log in or register
+>
+>   > ### {% icon comment %} Different Galaxy servers
+>   > The particular Galaxy server that you are using may look slightly different than the one shown in this training. Don't worry!
+>   {: .comment}
+>
+{: .hands_on}
+
+
+The Galaxy interface consists of three main parts:
+
+1. The available tools are listed on the left
+2. Your analysis history is recorded on the right
+3. The central panel will let you run analyses and view outputs
+
+![Galaxy ecosystem]({{ site.baseurl }}{% link shared/images/galaxy_interface.png %})
+
+
+# Create a history
+
+Galaxy allows you to create histories. Overall a history represents an experimental lab book, or a recipe very much like a cooking recipe with a list of ingredients (datasets) and a set of instructions 
+(pipeline of operations) that describes how to prepare or make something (such as a plot, or even a new dataset).
+The order of operations is important as very often the next operation takes as input the result of the previous operations. For instance, when baking
+a cake, you would first sift flour and then mix it with eggs as it would be impossible to sift flour afterwards.
+That is what we call a pipeline.
+
+Then the finalized pipeline can be serialized as a workflow. A workflow is the concatenation of one or multiple histories as a series of building blocks 
```

Comment:
You're right, Thanks for pointing it ! workflow isn't a concatenation of multiple histories. The word "histories" is unfortunate.
What the message could be is that executing all the steps in a specific way is the pipeline. This pipeline can be finalized and saved in Galaxy as a workflow so that it can be reused multiple times with a focus on datasets (not on the tools). That would be my bet. Maybe @annefou can bounce ?